### PR TITLE
Add all school details and basic styling

### PR DESCRIPTION
**Pull request by annamkummer**
The School component is displaying the data pulled from the API.

### CODE REVIEW

File: src/css/Results.css [hunk 1]
```diff
@@ -2,7 +2,8 @@
     text-align: center;
     display: flex;
     flex-direction: column;
-    height: 100vh;
+    background-color: #FAE9CD;
+    min-height: 100vh;
```
Comment (ewang0) [inline src/css/Results.css:6]: Nice adding min-height here! Good thinking to make it more dynamic.

File: src/css/School.css [hunk 2]
```diff
@@ -7,12 +7,49 @@
...
+    font-size: 2em;
+    color: #154874;
+}
+
+.detail {
+    text-align: left;
+}
+
+.bold {
+    font-weight: bold;
```
Comment (ewang0) [inline src/css/School.css:54]: Have you thought about refactoring to sass? Might cut down a few lines of code here.

File: src/components/Schools.js [hunk 3]
```diff
@@ -2,7 +2,6 @@ import School from './School'
 import '../css/Schools.css'
 
 const Schools = ({ schoolList }) => {
-    console.log(schoolList)
     let schools = (!schoolList.length) ? 
         'There are no schools that match that search. Please try again!' :
```
Comment (ewang0) [inline src/components/Schools.js:6]: Nice logic here to handle errors.

**Review by ewang0 — APPROVED** [review]

File: src/css/School.css (hunk 2 above)
Comment (annamkummer) [reply src/css/School.css:54]: Agreed, thanks! Will consider this for a future branch.

**Pull request merged**